features/get_task.feature

```diff
@@ -2,4 +2,7 @@ Feature: Get a task
 
   Scenario: User gets a task
     TODO: Add steps for getting a task
```

Comment:
TODO bitte löschen, sonst taucht das noch in der TODO Liste auf, obwohl adressiert.